configsnap

```diff
@@ -341,11 +341,13 @@ class dataGather:
                          (e.strerror))
             return None
 
-        if diff_proc.wait() != 0:
+        diff_stdout = diff_proc.communicate(input=None)
```

Comment:
Popen.communicate returns a tuple so does this need to be

```suggestion
        diff_stdout, diff_stderr = diff_proc.communicate(input=None)
```

Or adjust line 350 to

```python
            sys.stdout.writelines(diff_stdout[0])
```

Let me know if I've missed something.